ga4gh/cli.py

```diff
@@ -98,7 +98,7 @@ def setHttpClient(self, request, args):
         self._request = request
         self._verbosity = args.verbose
         self._httpClient = client.HTTPClient(
-            args.baseUrl, args.verbose, self._workarounds, self._key)
```

Comment:
I bet this is going to break a lot of tests, etc.  I actually am modifying the interface in #108 so it might be worth waiting until that change lands to revisit this issue